### DISCUSSION TITLE
Feature request: union of several lines into one

### DISCUSSION THREAD

**Discussion by stasDomb** [apache/superset#24233] [Ideas]
Hello! I would like to suggest providing additional feature in Superset. The feature looks simple and it's present in Tableau.
The example is in the screen
<img width="1104" alt="Screenshot 2023-05-25 at 10 22 09" src="https://github.com/apache/superset/assets/27229336/6eea50d6-423d-481b-bc18-fc68cd19388b">
<img width="1083" alt="Screenshot 2023-05-25 at 10 22 23" src="https://github.com/apache/superset/assets/27229336/49516976-cfa1-4ce7-99f4-729211475498">

Expected results:
selecting several lines -> combine them -> getting one common trend, which is sum of all lines.

It's really convenient and useful. What do you think?
Thanks

**Comment by rusackas** [comment]
It's an interesting idea... you can do this with more steps by creating/plotting an adhoc column or even adding it to the dataset as a calculated column. I assume when doing this you wouldn't want to _persist_ it, it would just be a user-level thing to temporarily fiddle with, like highlighting things by clicking the legend?

**Reply by stasDomb** [reply]
Hello. Thank you for your answer. You are right, it's possible this way, but it's not "user-friendly" in some cases. I mean, it's much-much easier to have it by 2-3 clicks on legend area

**Reply by stasDomb** [reply]
and when you have some distribution by some dynamic dimensions (which has name options), it is almost impossible

**Comment by stasDomb** [comment]
Hello! Maybe it's more relevant now?
